cf_xarray/accessor.py

```diff
@@ -1523,6 +1525,104 @@ def differentiate(
                 result *= -1
         return result
 
+    def add_canonical_attributes(
+        self,
+        override: bool = False,
+        skip: Union[str, List[str]] = None,
+        verbose: bool = False,
+        source=None,
+    ) -> Union[Dataset, DataArray]:
+        """
+        Add canonical CF attributes to variables with standard names.
+        Attributes are parsed from the official CF standard name table.
```

Comment:
Should link to http://cfconventions.org/standard-names.html and mention that an entry will added to the `history` attribute